WalletWasabi/Services/Terminate/TerminateService.cs

```diff
@@ -81,10 +98,13 @@ public void Terminate(int exitCode = 0)
 
 			AppDomain.CurrentDomain.ProcessExit -= CurrentDomain_ProcessExit;
 			Console.CancelKeyPress -= Console_CancelKeyPress;
+			SystemEvents.SessionEnding -= Windows_SystemEvents_SessionEnding;
```

Comment:
Should it be unsubscribed only for Windows?

Reply:
As far as I know, double or safety unsubscribe is fine and legit.

Reply:
Yeah, I meant more like to make it clear, it's Windows-only thing. Feel free to disregard that.

Reply:
ACK